docs/users/installation.md

```diff
@@ -0,0 +1,225 @@
+  settings in `.jvmopts`:
+
+      -Xss8m
+      -Xms1G
+      -Xmx4G
+
+### Verify installation
+
+To verify the installation, check that scalacOptions and libraryDependencies
+contain the values below.
+
+```scala
+> show scalacOptions
+[info] * -Yrangepos
+> show libraryDependencies
+[info] * org.scalameta:semanticdb-scalac:@SCALA212@:plugin->default(compile)
+```
+
+### Example project
+
+For a minimal example project using sbt-scalafix, see the
+[scalacenter/scalafix-sbt-example](https://github.com/scalacenter/scalafix-sbt-example)
+repository.
+
+```scala
+git clone https://github.com/olafurpg/scalafix-sbt-example
+cd scalafix-sbt-example
+sbt "scalafix RemoveUnusedImports"
+git diff // should produce a diff
+```
+
+### Settings and tasks
+
+| Name                    | Type           | Description                                                                                                                               |
+| ----------------------- | -------------- | ----------------------------------------------------------------------------------------------------------------------------------------- |
+| `scalafix <rule>..`     | `Unit`         | Run scalafix on project sources. See [Rules](rules/overview.md) or use tab completion to explore supported rules.                         |
+| `scalafixCli <args>`    | `Unit`         | Invoke scalafix command line interface directly. See {% doc_ref Installation, cli %} or use tab completion to explore supported arguments |
+| `scalafixTest <rule>..` | `Unit`         | Similar to `scalafix` except does not write to files, only reports an error if the task would produce a diff.                             |
+| `sbtfix <rule>..`       | `Unit`         | Run scalafix on the `*.sbt` and `project/*.scala` sources of this build. Only supports syntactic rules.                                   |
+| `sbtfixTest <rule>..`   | `Unit`         | Similar to `sbtfix` except does not write to files, only reports an error if the task would produce a diff.                               |
+| `scalafixConfig`        | `Option[File]` | .scalafix.conf file to specify which scalafix rules should run. Defaults to `.scalafix.conf` in the root directory if it exists.          |
+
+### FAQ
+
+- How to exclude/add files that are processed by the `scalafix` task?
+
+```
+unmanagedSources.in(Compile, scalafix) := unmanagedSources.in(Compile).value.filter(file => ...)
+```
+
+## Command line
+
+The recommended way to install the scalafix command-line interface is with
+[coursier](https://github.com/coursier/coursier#command-line).
+
+```sh
+// coursier
+coursier bootstrap ch.epfl.scala:scalafix-cli_@SCALA212@:@VERSION@ -f --main scalafix.cli.Cli -o scalafix
+./scalafix --help
+```
+
+### --help
+
+```scala mdoc:--help
+
+```
+
+## Maven
+
+It is possible to use scalafix with scala-maven-plugin but it requires a custom
+setup since there is no scalafix specific Maven plugin.
```

Comment:
Help wanted?

Reply:
Updated. Also improved the Maven section a bit more.